### ISSUE TITLE
Sampling regime metadata set incorrectly for Signature ADCP

### ISSUE THREAD

**Issue by mhidas** [aodn/imos-toolbox#554]
We have received [this file](https://github.com/aodn/imos-toolbox/files/3323062/IMOS_ANMN-WA_AETVZ_20181203T080004Z_WACA20_FV01_WACA20-1812-Signature250-194_END-20190620T023004Z_C-20190624T043041Z.nc.zip), from a "Norrtek Signature250" instrument, with the following global attributes:
```
                :instrument_burst_duration = 0. ;
                :instrument_burst_interval = 900. ;
                :instrument_sample_interval = -17173800. ;
```
The timestamps are at regular 15min intervals, so if it was originally burst sampled, the bursts have already been binned. In that case the `instrument_burst_` attributes should not be present (or at least set to correct values).

Either way, the `instrument_sample_interval` should be set to a sensible value (in this case 900.).

**Comment by sspagnol** [comment]
And while I'm a fan of [talk like a pirate day](https://en.wikipedia.org/wiki/International_Talk_Like_a_Pirate_Day) the instrument should be 'Nortek Signature250', i.e. only one 'r' in Nortek :)

**Comment by ocehugo** [comment]
I would expect a NaN first too. Nonetheless

Requirements:

- [ ] Before files are written they have to pass a schema.
- [ ] The schema has to check the metadata
- [ ] The schema checks for insturment_burst_duration is in seconds, it's smaller or equal to the sample interval or the burst interval.

**Comment by mhidas** [comment]
> The schema checks for insturment_burst_duration is in seconds, it's smaller or equal to the sample interval or the burst interval.

I'm not sure I fully understand what you're saying here. If the instrument is burst-sampling, the relative sizes of the attributes should be
`instrument_sample_interval <= instrument_burst_duration <= instrument_burst_interval`

If the instrument is _not_ burst-sampling then I think the `instrument_burst_` attributes should not even be there, to avoid confusion.

For data originating from a burst-sampled instrument, where bursts have already been averaged, these attributes can be left with values unchanged, so they still describe the _original sampling regime_ (as opposed to the timestamps in the present file). This is what our burst-averaged product is currently doing.

**Comment by ocehugo** [comment]
@mhidas, 
Agree. corrected

**Comment by ocehugo** [comment]
I got some files from Mark and the data was processed including the coordinate transformation and exported to mat files.

There are 1 ad2cp file and two mat files.

**Comment by ocehugo** [comment]
FYI: The reason why this data is with erroneous sampling metadata is that the data bypassed the signatureParser.m function (probably manually/custom handled).

The above parser does not accept the file as is - the "version" was not supported. Moreover, the logic that sets the sampling metadata logic didn't support the sampling scheme - a ```ping interval == burst interval == measurement average interval```

I change the requirements:

- [x] signatureParser is updated to handle the new version
- [x] signatureParser is updated to allow the sampling schema above
- [x] signature test files are updated on the test bucket.
- [x] Unit tests are provided.

**Comment by ocehugo** [comment]
not automatically closed when #623 changes were merged.
